staff/marti-malek/cv/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Martí Malek</h1>
+        <img src="superrandom" alt="My Image">
+        <h3>Personal Information</h3>
+        <address>
+        Address: C/Pedrell N.80 <b>Barcelona</b><br>
+        Tlf: 697 501 950<br>
+        e-mail: <a href="mailto:martimaledot@hotmail.es">martimalekdot@hotmail.es</a>
+        </address>
+        <h3>Profile</h3>
+        <p><i>Very hard-working person with energy and skills</i></p>
```

Comment:
This is to style it!

Use the <i> element only when there is not a more appropriate semantic element, such as:

<em> (emphasized text)
<strong> (important text)
<mark> (marked/highlighted text)
<cite> (the title of a work)
<dfn> (a definition term)

[https://www.w3schools.com/tags/tag_i.asp](url)

Reply:
Fixed it! 👍